### ISSUE TITLE
Impact Area editor has some odd behavior

### ISSUE THREAD

**Issue by codymccoy** [HydrologicEngineeringCenter/HEC-FDA#904]
The OK button is enabled to start with:
![image](https://github.com/HydrologicEngineeringCenter/HEC-FDA/assets/47757848/7f2d3eab-90fd-4c46-af14-284315872cec)

But if you click it, you get this message and then the ok button becomes not enabled:
![image](https://github.com/HydrologicEngineeringCenter/HEC-FDA/assets/47757848/d7ebd98e-1407-4bb5-8673-7a6b458cb8a1)

The file selector is still enabled which means that you could select a new file, but the unique column selector never shows up so I don't think you could ever save it with a new file. 
![image](https://github.com/HydrologicEngineeringCenter/HEC-FDA/assets/47757848/e6753dbb-efcb-41f9-917d-00a7cba4e83f)

You can't ever save this editor because there is no unique column visible.

**Comment by rnugent3** [comment]
So if imported, should we disable the OK button and then file selection button? we don't allow any editing so it makes sense that this screen would be mostly read only

**Comment by codymccoy** [comment]
Yeah, we could allow a name change, but the user can always right click the element and select "rename". We could just lock it all down and make it read only, or we could lock down the file selection and allow a name and description change. I think ideally we would allow the user to edit the name and description.

**Comment by rnugent3** [comment]
Let's allow the user to edit name and description. How much time do you anticipate this task will take?

**Comment by codymccoy** [comment]
1 to 2 hrs

**Comment by rnugent3** [comment]
Ok. Let's keep this in back log for now, saving our pennies for a rainy day...
